### PR TITLE
[Snyk] - Connect Snyk image to target

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

Deleted the previous guide that fills catalog images with snyk data to a new model that connects images to snyk target

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Fill catalog images with Snyk data
+# Connect images with Snyk target
 
-Snyk API provides information about scanned container images stored in container registries. This section aims to assist users in populating their software catalog using data obtained from the Snyk API. Follow the steps below to fill your catalog entities with images scanned by Snyk.
+Snyk API provides information about scanned images stored in container registries. This guide aims to assist users in populating their software catalog using data obtained from the Snyk API as well as connecting the image entities to their related Snyk targets.
```
Comment (phalbert) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md:3]: Snyk API provides information about scanned images stored in container registries. This guide demonstrates how to enrich your Port software catalog by importing image data from Snyk and establishing relationships between these images and their corresponding Snyk targets.

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Fill catalog images with Snyk data
+# Connect images with Snyk target
 
-Snyk API provides information about scanned container images stored in container registries. This section aims to assist users in populating their software catalog using data obtained from the Snyk API. Follow the steps below to fill your catalog entities with images scanned by Snyk.
+Snyk API provides information about scanned images stored in container registries. This guide demonstrates how to enrich your Port software catalog by importing image data from Snyk and establishing relationships between these images and their corresponding Snyk targets.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md:3]:
```suggestion
Snyk's API provides information about scanned images stored in container registries. This guide demonstrates how to enrich your Port software catalog by importing image data from Snyk and establishing relationships between these images and their corresponding Snyk targets.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Fill catalog images with Snyk data
+# Connect images with Snyk target
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-images-to-snyk-target.md:1]:
```suggestion
# Connect images to Snyk targets
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-service-to-target.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+<br/>
+
+### Create the service relation
+
+After installing the Snyk integration, your Port catalog will be synced with Snyk resources. Let's connect the Snyk target to our services. First, we will need to create a [relation](/build-your-software-catalog/customize-integrations/configure-data-model/relate-blueprints/) between our services and the corresponding Snyk target.
+
+1. Head back to the [Builder](https://app.getport.io/settings/data-model), choose the `service` <PortTooltip id="blueprint">blueprint</PortTooltip>, and click on `New relation`:
+
+<img src='/img/guides/githubServiceCreateRelation.png' width='60%' border="1px" />
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/guides/connect-service-to-target.md:24]: the images here are very big, make them a bit smaller

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**